packages/builders/scripts/testing/start-query-flows.js

```diff
@@ -0,0 +1,131 @@
+
+/**
+ * @import {QueryFlowsSF} from '@agoric/orchestration/test/fixtures/query-flows.contract.js';
+ */
+
+const contractName = 'queryFlows';
+const trace = makeTracer(contractName, true);
+
+/**
+ * See `@agoric/builders/builders/scripts/orchestration/init-query-flows.js` for
+ * the accompanying proposal builder. Run `agoric run
+ * packages/builders/scripts/orchestration/init-query-flows.js` to build the
+ * contract and proposal files.
+ *
+ * @param {BootstrapPowers} powers
+ */
+export const startQueryFlows = async ({
+  consume: {
+    agoricNames,
+    board,
+    chainStorage,
+    chainTimerService,
+    cosmosInterchainService,
+    localchain,
+    startUpgradable,
+  },
+  installation: {
+    // @ts-expect-error not a WellKnownName
+    consume: { [contractName]: installation },
+  },
+  instance: {
+    // @ts-expect-error not a WellKnownName
+    produce: { [contractName]: produceInstance },
+  },
+}) => {
+  trace(`start ${contractName}`);
+  await null;
+
+  const storageNode = await makeStorageNodeChild(chainStorage, contractName);
+  const marshaller = await E(board).getPublishingMarshaller();
+
+  /** @type {StartUpgradableOpts<QueryFlowsSF>} */
+  const startOpts = {
+    label: 'queryFlows',
+    installation,
+    terms: undefined,
+    privateArgs: await deeplyFulfilledObject(
+      harden({
+        agoricNames,
+        orchestrationService: cosmosInterchainService,
+        localchain,
+        storageNode,
+        marshaller,
+        timerService: chainTimerService,
+      }),
+    ),
+  };
+
+  const { instance } = await E(startUpgradable)(startOpts);
+  produceInstance.resolve(instance);
+};
+harden(startQueryFlows);
+
+export const getManifestForContract = (
+  { restoreRef },
+  { installKeys, ...options },
+) => {
+  return {
+    manifest: {
+      [startQueryFlows.name]: {
+        consume: {
+          agoricNames: true,
+          board: true,
+          chainStorage: true,
+          chainTimerService: true,
+          cosmosInterchainService: true,
+          localchain: true,
+          startUpgradable: true,
+        },
+        installation: {
+          consume: { [contractName]: true },
+        },
+        instance: {
+          produce: { [contractName]: true },
+        },
+      },
+    },
+    installations: {
+      [contractName]: restoreRef(installKeys[contractName]),
+    },
+    options,
+  };
+};
+
+/** @type {import('@agoric/deploy-script-support/src/externalTypes.js').CoreEvalBuilder} */
+export const defaultProposalBuilder = async ({ publishRef, install }) => {
+  return harden({
+    // Somewhat unorthodox, source the exports from this builder module
+    sourceSpec: '@agoric/builders/scripts/testing/start-query-flows.js',
+    getManifestCall: [
+      'getManifestForContract',
+      {
+        installKeys: {
+          queryFlows: publishRef(
+            install(
+              '@agoric/orchestration/test/fixtures/query-flows.contract.js',
```

Comment:
a little dicey since `test` tree isn't exported from that package. `@agoric/builders` is published to NPM and this will fail when installed from there (rather than the local tree).

I think a more strict package manager would also encounter this problem.

Please put in an exported path. Why not `src/examples`?

Reply:
`/examples` currently serves as a reference for prospective developers, and these particular invitations are not a pattern we want to teach - their only purpose is to facilitate walletDriver testing. In a recent [docs PR](https://github.com/Agoric/agoric-sdk/pull/9960/files#diff-a9fb1b44fc6e4cc2c108eae3cfda8db08a741638765efdab7c70a43a8482135fR5), they were interpreted as an example for sending queries despite the "this is for testing only" comments in the offer handlers.

I see two options - 
- move this to `/examples` and use the @file directive / comments to make it crystal clear this is not a pattern that should be used in production.
- make a new directory called `src/fixtures` where this can live

Reply:
sounds right. Either is fine with me. I'd go with `src/fixtures` to keep `src/examples` more browsable